scripts/copy_collection_files.py

```diff
@@ -2,17 +2,19 @@
 import json
 import os
 import shutil
-
+import io
 
 def copy_collection_file(dump_files, destination_prefix, working_dir):
-    file_log = open(os.path.join(working_dir, "files.log"), "w")
+    file_log = open(os.path.join(working_dir, "files.log"), "wb")
 
     for dump_file in dump_files:
-        with open(os.path.join(working_dir, dump_file), "r") as json_dump:
+        with open(dump_file, "r") as json_dump:
             data = json.load(json_dump)
             for record in data:
                 legacy_record_files = record["files"]
+                recid = record["recid"]
                 for legacy_record_file in legacy_record_files:
+                    print("Processing {}".format(recid))
```

Comment:
```suggestion

```

Reply:
this one was there on purpose, otherwise the script just goes on silently and the status in unknown